docs/api/internal.json

```
@@ -300,9 +300,14 @@
           },
           "status": {
             "type": "string",
-            "example": "publish",
+            "example": true,
```

I think the job status isn't modified by your code changes, right? It's still effectively the post status, no?

```suggestion
            "example": "publish",
```

---

Oops! It was an accidental change.